src/components/implementation/tests/micro_booter/micro_booter.c

```diff
@@ -568,7 +568,7 @@ term_fn(void *d)
 void
 cos_init(void)
 {
-	cos_meminfo_init(&booter_info.mi, BOOT_MEM_KM_BASE, COS_MEM_KERN_PA_SZ);
+	cos_meminfo_init(&booter_info.local_mi, BOOT_MEM_KM_BASE, COS_MEM_KERN_PA_SZ, BOOT_CAPTBL_SELF_LOCAL_PT);
```

Comment:
Because in the future, I want to two meminfo, local_mi and global_mi. I think this explicit distinction makes the code more clear.
In addition,  it seems global memory will become hot research topic and we may heavily use global memory in the near future. I am not sure local one is still typical or not.
But if you want to keep the name unchanged, it is totally fine.